src/models/ProjectSkillLower.js

```diff
@@ -0,0 +1,7 @@
+const ProjectSkillLower = 
+    `CREATE OR REPLACE TRIGGER ProjectSkillLower 
+    BEFORE INSERT OR UPDATE ON "ProjectSkill"
+    FOR EACH ROW
+    EXECUTE FUNCTION LowerInsert(skill)`;
+
+module.exports = ProjectSkillLower;
```

Comment:
Could this have been included in `ProjectSkill.js` as a named export? It's fine though.